layouts/partials/head.html

```diff
@@ -8,15 +8,22 @@
     name="description"
     content="{{if .IsHome}}{{$data.description | default $.Site.Data.config.description}}{{else}}{{.Summary}}{{end}}"
   />
-  <meta property="og:image" content="https://quartz.jzhao.xyz/icon.png">
+  <meta property="og:title" content="{{ .Title }}">
+  <meta property="og:type" content="website">
+  <meta property="og:image" content="icon.png">
+  <meta property="og:url" content="{{ .Permalink }}">
   <meta property="og:width" content="200">
   <meta property="og:height" content="200">
-  <meta property="og:type" content="website">
   <meta name="twitter:card" content="summary" />
-  <meta name="twitter:site" content="@_jzhao" />
   <meta name="twitter:title" content="{{ .Title }}" />
   <meta name="twitter:description" content="{{if .IsHome}}{{$data.description | default $.Site.Data.config.description}}{{else}}{{.Summary}}{{end}}" />
-  <meta name="twitter:image" content="https://quartz.jzhao.xyz/icon.png">
+  <meta name="twitter:image" content="icon.png">
+  {{ range $data.links }}
```

Comment:
@ jackyzha0 Is there a more elegant way to make the range loop? I tried to do it at the top and use the variable `$twitter_handle` here, but then the variable was always empty when using it here. If I ran the loop here, it works, but it could be more pretty.

Reply:
I used the static image, but I can't really test if it works in real life. It should as `icon.png` is in the `static` folder. I remember I had some issues when I used a relative path [here](https://github.com/sspaeti/second-brain-public/blob/1f8da0a9c7ca485f98ed0add83db86dc10bebf0e/layouts/partials/head.html#L21), therefore I used the full domain. But maybe that works now or I did something wrong. Should we try?

Reply:
I wonder if you can do `{{.Site.BaseURL}}icon.png`

Reply:
Oh yes, good input.

Reply:
@jackyzha0: just tagging it in case it went through the filter. I  think this PR would be ready to merge